alert/serializers.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models import Prefetch
+from rest_framework import serializers
+
+from alert.models import Registration
+from courses.models import Section
+from courses.serializers import MiniSectionSerializer
+
+
+class RegistrationSerializer(serializers.ModelSerializer):
+    section = MiniSectionSerializer(read_only=True)
```

Comment:
Might be better to have `section` be a `SlugRelatedField(slug_field='full_code')`. `full_code` is kinda new, but it will always hold the full code of a course or a section, all caps with dashes. you can see where it's set in the `save()` method on the `Course` and `Section` models. Means we don't need to do any extra lookups.